web/channels/player_channel.ex

```diff
@@ -34,7 +34,12 @@ defmodule Yggdrasil.PlayerChannel do
   end
 
   def handle_in("player_cmd", %{ "text" => text }, socket) do
-    Player.run_cmd(socket.assigns.user, Message.command(text))
-    {:noreply, socket}
+    case Parser.parse(text) do
+      {:ok, cmd} ->
+        Player.run_cmd(socket.assigns.user, cmd)
+        {:noreply, socket}
+      {:error, reason} ->
+        {:reply, {:error, Message.error(reason)},  socket}
```

Comment:
I didn't either till I typed in a command wrong. :smiley: and nothing crashed nor did I see anything anywhere until I looked at WS payload in the browser debugger.